xbmc/FileItem.cpp

```diff
@@ -124,7 +124,7 @@ void CFileItem::FillMusicInfoTag(const CPVRChannelPtr& channel, const CPVREpgInf
       musictag->SetGenre(tag->Genre());
       musictag->SetDuration(tag->GetDuration());
     }
-    else if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(CSettings::SETTING_EPG_HIDENOINFOAVAILABLE))
```

Comment:
Two reasons.  The default Title for the music FileItem is pvr....pvr which makes no sense when used as a music title so this is a reasonable default.  Even without the change to display Shoutcast data when the hide setting is false PVR radio displays the pvr...pvr text while tuning between channels when there is no EPG data until play begins.

Reply:
Makes no sense tbh; seems only to workaround the real cause of your problem.

Reply:
Perhaps looking at the original code will explain the problem better.  It is saying 

if "hide not available" setting is true
set title to "not available"

which the reverse of what we want but it gets corrected later by PVRInfo.

Reply:
Ah, now I see it. Bur shouldn't the right fix then to introduce the obviously missing negation to fix the logic? And how is your change related to this PR?

Reply:
If you follow it through that odd logic was likely written to keep pvr...pvr from appearing when the the setting was true.  My chance was to issue the 19055 for both cases.  Unfortunately I can't set it at as empty because that is the same as default and pvr....pvr shows.  I guess I could pass a space but that seems a bit of a hack.

It relates to this PR because if I want to use Shoutcast data and optionally MusicPlayer.Title in a skin I would prefer 19055 or  even  " " to pvr....pvr the the default FileItem Title.

Reply:
The real question seems to be why the pvr...pvr title is chosen as default. I cannot imagine what this should be good for. Maybe this is something  that needs fixing?

Reply:
Perhaps I confused this a bit with my short name.  What is being displayed is the channel filename ie from pvr://channels/radio/All channels/pvr.nextpvr_9016.pvr  I see pvr.nextpvr_9016.pvr  For a music file that makes sense when no other information is available.   My preference is  a space " " but I used the 19055 since that is what was being used before.  I could also use the channel name, but the function is already overboard populating it.

Reply:
> My preference is a space " "

 hack.

>  I could also use the channel name

  this sounds correct.

Reply:
I still think this change is not correct. See what I wrote in previous comments. Why is default title like pvr.nextpvr_9016.pvr? This must be fixed.